app/views/users/show/_confirm_outgoing_swap.html.haml

```diff
@@ -51,7 +51,7 @@
   we'll cancel the swap and you can pick someone else.
 
 %h3.text-center
-  In the meantime ...
+  In the meantime…
```

Comment:
We shouldn't really have a space before an ellipsis, use an ellipsis char rather than three fullstops.